sample.env

```diff
@@ -139,6 +139,9 @@ PAGINATION_NUMBER=25
 # Default is set to 10 rows
 NUMBER_OF_ROWS=10
 
+# Specify if you want to display the Google Calendar button
+ENABLE_GOOGLE_CALENDAR_BUTTON=true
```

Comment:
This should be disabled by default

Reply:
Agreed.

Reply:
Maybe it could have been written it defaults to false, but now it's been merged I think it's okay to keep it this way.